### PR TITLE
Schema refinements

### LINKED ISSUE #16: Schema refinement
we should refine which parts of the Schema are actually nullable and which parts are not. This should be done before integrating the UI since it makes it easier if the types are working.

### PR DESCRIPTION
**Pull request by RohrerF**
This PR closes #16 

TODOs:

- [x] snapshot test of schema
- [x] mutation input types

### CODE REVIEW

**Comment by RohrerF** [comment]
@maracuja-juice Open the [Snapshot](https://github.com/ultimate-ttt/ultimate-ttt-backend/blob/2e1bb46e0cd454809f07a017b334b82e9a2c8d01/src/Api.Tests/__snapshots__/SchemaBuilderTests.Ensure_Schema_IsCorrect.snap) to see the current Schema

File: .gitignore [hunk 1]
```diff
@@ -4,10 +4,6 @@
 
 ### Code ###
 .vscode/*
```
Comment (panmona) [inline .gitignore:6]: image
What changed your opinion on this?

File: src/Api/Types/SchemaBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+namespace UltimateTicTacToe.Api.Types
+{
+    public static class SchemaBuilder
+    {
+        public static Schema BuildSchema()
+        {
+            return Schema.Create(c =>
+            {
+                //GraphQL Types
```
Comment (panmona) [inline src/Api/Types/SchemaBuilder.cs]: add a space

**Review by panmona — DISMISSED** [review]

**Comment by panmona** [comment]
The non nullable additions will probably help in integrating this.

File: .gitignore (hunk 1 above)
Comment (RohrerF) [reply .gitignore:6]: found out that the current setting are about the same as the default ==> don't need the in git

Comment (panmona) [reply .gitignore:6]: :)

**Review by panmona — APPROVED** [review]

**Pull request merged**